packages/jitm/src/JITM.php

```diff
@@ -436,7 +436,8 @@ function get_messages( $message_path, $query ) {
 			)
 		);
 
-		$user = wp_get_current_user();
+		$user       = wp_get_current_user();
+		$user_roles = implode( ',', $user->roles );
```

Comment:
What do you think about moving this until after the `if ( ! $user ) {` check, to avoid any errors?

Reply:
I think that's a good idea! Thanks for catching that!